### PR TITLE
chore: nepali translations and remove unused keys

### PR DESCRIPTION
**Pull request by 1yuv**
# Description

This PR addresses multiple issues and addresses some translation fixes.
- Add nepali translations for all keys and correct any misleading translations #9028
- Fix mistake with french translations #9384 
- Correct sentence ending for hindi translations 
- Remove unused translations keys from all languages

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:ne_translations_201_800/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:ne_translations_201_800/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:ne_translations_201_800/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: api/resources/translations/messages-ne.properties [hunk 1]
```diff
@@ -279,34 +279,35 @@ Visits = स्वाथ्य संस्थामा जाँच
...
+admin.message.queue.status = अवस्था
+admin.message.queue.due = पठाउने समय
+admin.message.queue.updated = अन्तिम अपडेट
+admin.message.queue.view.report = रिपोर्ट हेर्ने
+admin.pagination.detail = कुल {{total}} मध्ये {{first}} - {{last}} सम्मको देखाइएको
+admin.pagination.first = पहिलो
+admin.pagination.last = अन्तिम
+admin.pagination.next = अर्को
+admin.pagination.prev = पछिल्लो
+admin.targets.description = उद्धेश्य नराखिएको लक्षमा -१ देखाइने छ
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:310]:
```suggestion
admin.targets.description = उद्धेश्य नराखिएको लक्ष्यमा -१ देखाइने छ
```

File: api/resources/translations/messages-ne.properties [hunk 2]
```diff
@@ -624,19 +638,41 @@ email.invalid =
...
 enketo.drawwidget.drawing = चित्र
 enketo.drawwidget.signature = हस्ताक्षर
 enketo.form.required = 
-error.403.description = तपाईं यस पृष्ठ हेर्न अपर्याप्त विशेषाधिकार छ । आफ्नो विशेषाधिकार वृद्धि गर्न प्रशासकीय कुरा ।
-error.403.title = त्रुटी शिर्षक
-error.404.description = के देख थिए पृष्ठ फेला परेन ।
+enketo.filepicker.file = फाइल
+enketo.filepicker.placeholder = फाइल अपलोड गर्नलाई यहाँ थिच्नुहोस्। (< {{maxSize}})
+enketo.filepicker.notFound = {{existing}} फाइल भेटिएन। (तपाईंले पहिले नै बुझाइसकेको फाइल राख्ने भए यत्तिकै छाड्नुहोस्।)
+enketo.filepicker.waitingForPermissions = Waiting for user permissions.
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:649]:
```suggestion
enketo.filepicker.waitingForPermissions = प्रयोगकर्ता अनुमतिहरूको लागि पर्खँदै।
```

File: api/resources/translations/messages-ne.properties [hunk 3]
```diff
@@ -1027,6 +1105,7 @@ report.delivery.pregnancy_outcome_label =
 report.delivery.title.sms = 
 report.edit = सम्पादन रिपोर्ट
 report.flag.title.sms = 
+report.new_report.button =
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1108]:
```suggestion
report.new_report.button = नयाँ रिपोर्ट
```

File: api/resources/translations/messages-ne.properties [hunk 4]
```diff
@@ -1027,6 +1105,7 @@ report.delivery.pregnancy_outcome_label =
 report.delivery.title.sms = 
 report.edit = सम्पादन रिपोर्ट
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1106]:
```suggestion
report.edit = रिपोर्ट सम्पादन
```

File: api/resources/translations/messages-ne.properties [hunk 5]
```diff
@@ -1156,6 +1244,7 @@ schedule.registration_anc =
 schedule.registration_anc_pnc = 
 scheduled = समयतालिका
 scheduled_tasks = कार्यतालिका
+search_bar.filter.label = फिल्टर
 select.mode.count.plural =
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1248]:
```suggestion
select.mode.count.plural = {{number}} वटा रेकर्ड छानिएको
```

File: api/resources/translations/messages-ne.properties [hunk 6]
```diff
@@ -1156,6 +1244,7 @@ schedule.registration_anc =
 schedule.registration_anc_pnc = 
 scheduled = समयतालिका
 scheduled_tasks = कार्यतालिका
+search_bar.filter.label = फिल्टर
 select.mode.count.plural = 
 select.mode.count.singular =
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1249]:
```suggestion
select.mode.count.singular = एउटा रेकर्ड छानिएको
```

File: api/resources/translations/messages-ne.properties [hunk 7]
```diff
@@ -1156,6 +1244,7 @@ schedule.registration_anc =
 schedule.registration_anc_pnc = 
 scheduled = समयतालिका
 scheduled_tasks = कार्यतालिका
+search_bar.filter.label = फिल्टर
 select.mode.count.plural = 
 select.mode.count.singular = 
 select.mode.delete.all =
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1250]: सबै रेकर्ड मेटाउने

File: api/resources/translations/messages-ne.properties [hunk 8]
```diff
@@ -1165,11 +1254,9 @@ select.mode.start =
 select.mode.stop =
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1254]:
```suggestion
select.mode.stop = बन्द गर्ने
```

File: api/resources/translations/messages-ne.properties [hunk 9]
```diff
@@ -1165,11 +1254,9 @@ select.mode.start =
 select.mode.stop = 
 select\ all = सबै छान्नुहोस्
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1255]:
```suggestion
select\ all = सबै छान्ने
```

File: api/resources/translations/messages-ne.properties [hunk 10]
```diff
@@ -1272,14 +1346,17 @@ task.type = कार्य प्रकार
...
 task.warning.high_risk = 
 task.warning.home_birth = 
-tasks = पठाइएको संदेश
+tasks = पठाइएको सन्देश
 tasks.0.messages.0.message = सन्देश
 tasks.0.messages.0.to = पाउने
-tasks.0.state = राज्य
+tasks.0.state = 
 tasks.0.timestamp = समय/ मिति
 tasks.disabled =
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1354]:
```suggestion
tasks.disabled = यस प्रयोगकर्ताका लागि कार्यहरू उपलब्ध गराइएको छैन।
```

**Review by binokaryg — COMMENTED** [review]
I have added some translations and requested some changes.

**Review by garethbowen — CHANGES_REQUESTED** [review]

File: api/resources/translations/messages-ne.properties [hunk 11]
```diff
@@ -446,513 +454,582 @@ confirm.destructive.navigation.forms = यो फारम समाप्त 
...
-horti.stage.warmingViews = 
+horti.stage.deploying = नयाँ इन्स्टलेशन राख्दै
+horti.stage.download = डाउनलोड र इन्स्टलको लागि स्टेज गर्दै
+horti.stage.extractingDdocs = ddocs निकाल्दै
+horti.stage.init = Horticulturalist सुरु गर्दै
+horti.stage.initDeploy = डिप्लोयमेन्ट सुरु गर्दै
+horti.stage.postCleanup = डिप्लोयमेन्ट पछिको तयारि, इन्स्टल पुरा भयो
+horti.stage.preCleanup = डिप्लोयमेन्ट अघिको तयारि
+horti.stage.readyToDeploy = Views warming पूरा, डिप्लोयमेन्टको लागि तयार भयो
+horti.stage.warmingViews = Views warming गर्दै
```
Comment (garethbowen) [inline api/resources/translations/messages-ne.properties:737]: I bet these aren't used any more either.

File: api/resources/translations/messages-ne.properties [hunk 12]
```diff
@@ -989,365 +1066,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
-reports.verify.valid = 
+reporter.link = मेडिक रिपोर्टरमा जानुहोस् 
+reporter.section.help = SMS फर्महरूको परीक्षण गर्न मेडिक रिपोर्टर प्रयोग गर्नुहोस्।
+reporter.section.title = मेडिक रिपोर्टर
+reporting_unit_not_found = रिपोर्टिङ् युनिटको आईडी मिलेन। कृपया ​मिलाएर​ पुन\:​ पठाउनुहोला।
+reports.none = कुनै रिपोर्ट फेला परेन
+reports.none.n.months = {MONTHS, plural, one{पछिल्लो महिनामा कुनै रिपोर्टहरू छैनन्।} other{पछिल्लो \# महिनामा कुनै रिपोर्टहरू छैनन्।}}
+reports.verify = समीक्षा गर्नुहोस्
+reports.verify.valid = सही छ
+reports.verify.invalid = त्रुटीपूर्ण छ
```
Comment (garethbowen) [inline api/resources/translations/messages-ne.properties:1152]: Alphabetize this.

**Review by garethbowen — APPROVED** [review]
Minor suggestions inline.

**Review by binokaryg — COMMENTED** [review]
Added suggestions until line 105.

File: api/resources/translations/messages-ne.properties [hunk 13]
```diff
@@ -963,22 +1032,22 @@ person.field.national_id_number = राष्ट्रीय आईडी न
...
-quarter.plural = 
+place.deleted = स्थान मेटाइयो
+place.unavailable = स्थान नभेटिएको
+privacy.policy = गोपनियता नीति
+privacy.policy.accept = गोपनियता नीति मान्य छ।
+privacy.policy.not.found = गोपनियता नीति भेटिएन 
+purge.description = Purge को लागि योग्य डकुमेन्टहरु थाहा पाउन तत्काल ट्रिगर गर्नुहोस्।
+purge.title = Purge
+quarter = त्रैमाशिक
+quarter.plural = त्रैमाशिकहरु
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1045]:
```suggestion
quarter = त्रैमासिक
quarter.plural = त्रैमासिकहरू
```

File: api/resources/translations/messages-ne.properties [hunk 14]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
-reports.verify.invalid = 
-reports.verify.valid = 
+reporter.link = मेडिक रिपोर्टरमा जानुहोस् 
+reporter.section.help = SMS फर्महरूको परीक्षण गर्न मेडिक रिपोर्टर प्रयोग गर्नुहोस्।
+reporter.section.title = मेडिक रिपोर्टर
+reporting_unit_not_found = रिपोर्टिङ् युनिटको आईडी मिलेन। कृपया ​मिलाएर​ पुन\:​ पठाउनुहोला।
+reports.none = कुनै रिपोर्ट फेला परेन
+reports.none.n.months = {MONTHS, plural, one{पछिल्लो महिनामा कुनै रिपोर्टहरू छैनन्।} other{पछिल्लो \# महिनामा कुनै रिपोर्टहरू छैनन्।}}
+reports.verify = समीक्षा गर्नुहोस्
+reports.verify.invalid = त्रुटीपूर्ण छ
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1143]:
```suggestion
reports.verify.invalid = त्रुटिपूर्ण छ
```

File: api/resources/translations/messages-ne.properties [hunk 15]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
-schedule.pnc_from_app = 
-schedule.registration_anc = 
-schedule.registration_anc_pnc = 
+schedule.anc_lmp = गर्भजाँच सम्झना तालिका\: समूह {{group}}
+schedule.anc_lmp_from_app = गर्भजाँच सम्झना तालिका\: समूह {{group}}
+schedule.anc_no_lmp = गर्भजाँच सम्झना तालिका\: समूह {{group}}
+schedule.child_health = बाल स्वास्थ्य\: समूह {{group}}
+schedule.delivery_lmp = प्रसुति समयतालिका\: समूह {{group}}
+schedule.delivery_lmp_from_app = प्रसुति समयतालिका\: समूह {{group}}
+schedule.delivery_no_lmp = प्रसुति समयतालिका\: समूह {{group}}
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1154]:
```suggestion
schedule.delivery_lmp = प्रसूति समयतालिका\: समूह {{group}}
schedule.delivery_lmp_from_app = प्रसूति समयतालिका\: समूह {{group}}
schedule.delivery_no_lmp = प्रसूति समयतालिका\: समूह {{group}}
```

File: api/resources/translations/messages-ne.properties [hunk 16]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+schedule.imm = खोप\: समूह {{group}}
+schedule.nutrition_evaluation = पोषण मुल्यांकन
+schedule.nutrition_otp = बाह्य रोगोपचार कार्यक्रम (OTP) तालिका
+schedule.nutrition_sc = स्थिरीकरण केन्द्र (SC) तालिका
+schedule.nutrition_sc_exit = स्थिरीकरण केन्द्र बाट निस्कने तालिका
+schedule.nutrition_sfp = पुरक खानपान कार्यक्रम (SFP) तालिका
+schedule.pnc = प्रसुतिपछिको हेरचाह\:समूह {{group}}
+schedule.pnc_facility_birth = स्वास्थ्य संस्थामा प्रसुतिपछिको हेरचाह\:समूह {{group}}
+schedule.pnc_from_app = एपबाट प्रसुतिपछिको हेरचाह\:समूह {{group}}
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1163]:
```suggestion
schedule.nutrition_sfp = पूरक खानपान कार्यक्रम (SFP) तालिका
schedule.pnc = प्रसूतिपछिको हेरचाह\:समूह {{group}}
schedule.pnc_facility_birth = स्वास्थ्य संस्थामा प्रसूतिपछिको हेरचाह\:समूह {{group}}
schedule.pnc_from_app = एपबाट प्रसूतिपछिको हेरचाह\:समूह {{group}}
```

File: api/resources/translations/messages-ne.properties [hunk 17]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
-select.mode.count.plural = 
-select.mode.count.singular = 
-select.mode.delete.all = 
-select.mode.deselect.all = 
-select.mode.select.all = 
-select.mode.start = 
-select.mode.stop = 
+search_bar.filter.label = फिल्टर
+select.mode.count.plural = {{number}} रेकर्ड छानिएका छन्
+select.mode.count.singular = १ रेकर्ड छानिएको छ
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1170]:
```suggestion
select.mode.count.singular = एउटा रेकर्ड छानिएको छ
```

File: api/resources/translations/messages-ne.properties [hunk 18]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
-status.sms.valid = 
-submit.icon = 
-sync.last_success = 
-sync.now = 
-sync.status.in_progress = 
-sync.status.not_required = 
-sync.status.required = 
-sync.status.unknown = 
+status.review.correct = समीक्षा गरिएको : सही  
+status.review.errors = समीक्षा गरिएको : त्रुटीपूर्ण
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1207]:
```suggestion
status.review.errors = समीक्षा गरिएको : त्रुटीपूर्ण
```
```suggestion
status.review.errors = समीक्षा गरिएको : त्रुटिपूर्ण
```

File: api/resources/translations/messages-ne.properties [hunk 19]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
-task.postnatal_danger_sign.title = 
-task.postnatal_home_birth.title = 
-task.postnatal_missing_visit.title = 
-task.pregnancy_danger_sign.title = 
-task.pregnancy_missing_birth.title = 
-task.pregnancy_missing_visit.title = 
+task.days.left = {DAYS, plural, one{१ दिन बाँकी} other{\# दिन बाँकी}}
+task.list.complete = अरु कार्यहरु छैनन्
+task.overdue = अहिले गर्नुपर्ने
+task.overdue.days = {DAYS, plural, =0{आज गर्नुपर्ने} =1{हिजो गर्नुपर्ने} other{Due \# दिन अगाडी नै गर्नुपर्ने }}
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1239]:
```suggestion
task.overdue.days = {DAYS, plural, =0{आज गर्नुपर्ने} =1{हिजो गर्नुपर्ने} other{\# दिन अगाडी नै गर्नुपर्ने }}
```

File: api/resources/translations/messages-ne.properties [hunk 20]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
-user.phone.help = 
-user.place.help = 
-user.username.help = 
-username.invalid = 
-username.taken = 
-usertype._admin = 
+training_cards.error.loading = तालिम लोड गर्दा त्रुटी भयो। तपाइँको सुपरभाइजरलाई सम्पर्क गर्नुहोस्। 
+training_cards.error.save = तालिम सेभ गर्न त्रुटी।
+training_cards.form.saved = तालिम सम्पन्न भयो। 
+training_cards.modal.title = महत्वपुर्ण परिवर्तनहरु
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1267]:
```suggestion
training_cards.modal.title = महत्वपूर्ण परिवर्तनहरू
```

File: api/resources/translations/messages-ne.properties [hunk 21]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+translation.add = नयाँ अनुवाद कुञ्जी थप्नुहोस्
+translation.key = अनुवाद कुञ्जी
+unique.id = आईडी
+unknown.contact = अपरिचिती सम्पर्क
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1271]:
```suggestion
unknown.contact = अपरिचित सम्पर्क 
```

File: api/resources/translations/messages-ne.properties [hunk 22]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+training_cards.error.loading = तालिम लोड गर्दा त्रुटी भयो। तपाइँको सुपरभाइजरलाई सम्पर्क गर्नुहोस्। 
+training_cards.error.save = तालिम सेभ गर्न त्रुटी।
+training_cards.form.saved = तालिम सम्पन्न भयो। 
+training_cards.modal.title = महत्वपुर्ण परिवर्तनहरु
+translation.add = नयाँ अनुवाद कुञ्जी थप्नुहोस्
+translation.key = अनुवाद कुञ्जी
+unique.id = आईडी
+unknown.contact = अपरिचिती सम्पर्क 
+upgrade = अपग्रेड
+upgrade.description = तपाईँको एपलाई कुनै रिलीज, बीटा, वा ब्रान्चमा अपग्रेड गर्न, पहिलो चरणको रूपमा स्टेज गर्न सिफारिस गरिन्छ। यसले प्रयोगकर्ताहरूलाई अवरोध नगरी पृष्ठभूमिमा इन्स्टलेसन तयारी गर्छ। स्टेजिंग पुरा भएपछि, अपग्रेडको लागि 'Install' क्लिक गर्नुहोस्। यो पछि, यसलाई पहिलाकै अवस्थामा  सकिँदैन, त्यसैले कृपया तपाईँको डेटा ब्याकअप गरिएको छ र तपाईँका प्रयोगकर्ताहरूलाई डाउनटाइमको जानकारी दिइएको छ भन्ने निश्चित गर्नुहोस्।
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1273]:
```suggestion
upgrade.description = तपाईँको एपलाई कुनै रिलीज, बेटा, वा शाखामा अपग्रेड गर्न, पहिलो चरणको रूपमा स्टेज गर्न सिफारिस गरिन्छ। यसले प्रयोगकर्ताहरूलाई अवरोध नगरी पृष्ठभूमिमा इन्स्टलेसन तयारी गर्छ। स्टेजिंग पुरा भएपछि, अपग्रेडको लागि 'इन्स्टल गर्नुहोस्' मा क्लिक गर्नुहोस्। त्यस पछि, यसलाई पहिलाकै अवस्थामा लैजान सकिँदैन, त्यसैले कृपया तपाईँको डेटा ब्याकअप गरिएको छ र तपाईँका प्रयोगकर्ताहरूलाई सेवा अवरुद्ध रहने बारे जानकारी दिइएको छ भन्ने निश्चित गर्नुहोस्।
```

File: api/resources/translations/messages-ne.properties [hunk 23]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+upload = अपलोड
+upload.favicon = आइकन
+upload.header.logo = शिर्षक (header) को लोगो
+upload.icon = आइकन अपलोड गर्ने
+upload.json.forms.help = फाइल फारम परिभाषा एक JSON इनकोडिंग सुची समावेश गर्नुपर्छ । पहिल्यै भएका कुनै कुरा अध्यवाधिक गरिने छ।
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1278]:
```suggestion
upload.json.forms.help = फाइलमा JSON इनकोड गरिएको फारम परिभाषाको सूची समावेश हुनुपर्छ । एक पटकमा एउटा मात्र फारम फाइल अपलोड गर्न मिल्नेछ र पहिल्यै भएका फारमहरू फेरिने छन्।
```

File: api/resources/translations/messages-ne.properties [hunk 24]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+upload.json.forms.title = JSON फर्महरू अपलोड गर्नुहोस् 
+upload.sms.forms = SMS फर्महरू अपलोड गर्नुहोस् 
+upload.xform.context = XForm सन्दर्भ अपलोड गर्नुहोस् 
+upload.xform.help = XForm अपलोड सहायता
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1282]:
```suggestion
upload.xform.help = फाइलमा XForm र त्यससँग सम्बन्धित JSON सन्दर्भ समावेश हुनुपर्छ। पहिल्यै भएका फारमहरू फेरिने छन्।
```

File: api/resources/translations/messages-ne.properties [hunk 25]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+upload.xform.meta.help = XForm मेटाडाटा अपलोड सहायता
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1283]:
```suggestion
upload.xform.meta.help = आइकन, शीर्षक, सन्दर्भ, र यस्तै अन्य सहित JSON ढाँचामा मेटा फाइल।
```

File: api/resources/translations/messages-ne.properties [hunk 26]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+upload.xform.title = XForm अपलोड गर्नुहोस्
+upload.xform.xml = XForm XML अपलोड गर्नुहोस्
+upload.xform.xml.help = XForm XML अपलोड सहायता
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1286]:
```suggestion
upload.xform.xml.help = XML मा XForm परिभाषा।
```

File: api/resources/translations/messages-ne.properties [hunk 27]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+url = URL
+user.fullname.help = प्रयोगकर्ताको पूरा नाम
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1288]:
```suggestion
user.fullname.help = हाल एपमा कतै देखाइएको छैन।
```

File: api/resources/translations/messages-ne.properties [hunk 28]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+user.password.current = अहिलेको पासवर्ड
+user.phone.help = प्रयोगकर्ताको फोन सहायता
+user.place.help = प्रयोगकर्ताको स्थान सहायता
+user.username.help = प्रयोगकर्ताको प्रयोगकर्ता नाम सहायता
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1292]:
```suggestion
user.phone.help = ड्यास वा विराम चिन्ह बिना मान्य फोन नम्बर हुनुपर्छ।
user.place.help = सबै प्रयोगकर्ताहरू सही रूपमा अवस्थित हुन र लग इन गर्दा उपयुक्त डाटा हेर्नको लागि सही स्थानसँग सम्बन्धित हुनुपर्छ।
user.username.help = यो तपाईले एपमा लग इन गर्न प्रयोग गर्नुहुनेछ।
```

File: api/resources/translations/messages-ne.properties [hunk 29]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+username.invalid = अमान्य प्रयोगकर्ता नाम
+username.taken = प्रयोगकर्ता नाम लिइएको छ
+users.import.accepted_types = स्वीकार गरिएका प्रकारहरू आयात गर्नुहोस्
+users.import.cancel_upload = अपलोड रद्द गर्नुहोस्
+users.import.confirm_upload = अपलोड पुष्टि गर्नुहोस्
+users.import.confirm_upload_text = अपलोड पुष्टि पाठ
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1298]:
```suggestion
users.import.confirm_upload_text = के तपाई साँच्चै अपलोड गर्न चाहनुहुन्छ?
```

File: api/resources/translations/messages-ne.properties [hunk 30]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+users.import.instructions = आयात निर्देशनहरू
+users.import.processing.title = आयात प्रशोधन शीर्षक
+users.import.processing.instructions = आयात प्रशोधन निर्देशनहरू
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1301]:
```suggestion
users.import.processing.instructions = धेरै प्रयोगकर्ताहरू थप्न, कृपया उपयुक्त स्प्रिडसिट अपलोड गर्नुहोस्।
```

File: api/resources/translations/messages-ne.properties [hunk 31]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+upload.xform.xml.help = XForm XML अपलोड सहायता
+url = URL
+user.fullname.help = प्रयोगकर्ताको पूरा नाम
+user.password.current = अहिलेको पासवर्ड
+user.phone.help = प्रयोगकर्ताको फोन सहायता
+user.place.help = प्रयोगकर्ताको स्थान सहायता
+user.username.help = प्रयोगकर्ताको प्रयोगकर्ता नाम सहायता
+username.invalid = अमान्य प्रयोगकर्ता नाम
+username.taken = प्रयोगकर्ता नाम लिइएको छ
+users.import.accepted_types = स्वीकार गरिएका प्रकारहरू आयात गर्नुहोस्
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1295]:
```suggestion
users.import.accepted_types = .csv फाइल मात्र स्वीकार गरिन्छ।
```

File: api/resources/translations/messages-ne.properties [hunk 32]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+upload.xform.xml = XForm XML अपलोड गर्नुहोस्
+upload.xform.xml.help = XForm XML अपलोड सहायता
+url = URL
+user.fullname.help = प्रयोगकर्ताको पूरा नाम
+user.password.current = अहिलेको पासवर्ड
+user.phone.help = प्रयोगकर्ताको फोन सहायता
+user.place.help = प्रयोगकर्ताको स्थान सहायता
+user.username.help = प्रयोगकर्ताको प्रयोगकर्ता नाम सहायता
+username.invalid = अमान्य प्रयोगकर्ता नाम
+username.taken = प्रयोगकर्ता नाम लिइएको छ
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1294]:
```suggestion
username.taken = प्रयोगकर्ता नाम "{{username}}" पहिले नै लिइएको छ।
```

File: api/resources/translations/messages-ne.properties [hunk 33]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+users.import.summary.failed_users = असफल प्रयोगकर्ताहरू
+users.import.summary.here = यहाँ
+users.import.summary.previously_added_users = पहिले थपिएका प्रयोगकर्ताहरू
+users.import.summary.resubmit.instructions = पुन: सबमिट निर्देशनहरू
+users.import.summary.resubmit.step1 = १. डाउनलोड गरिएको फाइल खोल्नुहोस्
+users.import.summary.resubmit.step2 = २. तपाईँको फर्म्याट गरिएको स्प्रेडशीटसँग त्रुटिहरूलाई तुलना गर्नुहोस्
+users.import.summary.resubmit.step3 = ३. त्रुटिहरू सच्याउनुहोस् र .csv मा निर्यात गर्नुहोस्
+users.import.summary.resubmit.step4 = ४. अपडेट गरिएको स्प्रेडशीट पुनः अपलोड गर्नुहोस्
+users.import.title = प्रयोगकर्ता आयात शीर्षक
+users.import.unavailable.instructions = अनुपलब्ध निर्देशनहरू
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1314]:
```suggestion
users.import.unavailable.instructions = प्रणालीमा अरू कोही हाल प्रयोगकर्ताहरू थप्ने प्रक्रियामा छन्।
```

File: api/resources/translations/messages-ne.properties [hunk 34]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+users.import.unavailable.retry = पुन: प्रयास गर्नुहोस्
+users.import.unavailable.title = अनुपलब्ध शीर्षक
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1316]:
```suggestion
users.import.unavailable.title = उपलब्ध छैन
```

File: api/resources/translations/messages-ne.properties [hunk 35]
```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
...
+users.import.upload_button = अपलोड बटन
```
Comment (binokaryg) [inline api/resources/translations/messages-ne.properties:1317]:
```suggestion
users.import.upload_button = अपलोड .csv
```

**Review by binokaryg — CHANGES_REQUESTED** [review]
More changes requested.

Does every `:` require escaping (`\:`)? If so, we have missed to escape many such texts.

File: api/resources/translations/messages-ne.properties (hunk 22 above)
Comment (1yuv) [reply api/resources/translations/messages-ne.properties:1273]: शाखा doesn't give a good meaning here and it's not the right translation, so keeping branch as that makes it more meaningful.

File: api/resources/translations/messages-ne.properties (hunk 27 above)
Comment (1yuv) [reply api/resources/translations/messages-ne.properties:1288]: There is wrong translation in the english for this key, and we shouldn't translate exactly that based on how it's used. I'll go ahead and update the english and other translation meaning as well.

File: api/resources/translations/messages-ne.properties (hunk 22 above)
Comment (binokaryg) [reply api/resources/translations/messages-ne.properties:1273]: That is what you have added. I'm just trying to make it consistent:
https://github.com/medic/cht-core/blob/17b69b1bae970ae5bbacf2693b61d007a859485a/api/resources/translations/messages-ne.properties#L763

File: api/resources/translations/messages-ne.properties [hunk 36]
```diff
@@ -227,48 +227,48 @@ Send = पठाउनुहोस्
...
+Translation\ file\ help = यो भाषा लागि अनुवाद प्रतिस्थापन गर्न आयात गर्न .properties फाइल चयन गर्नुहोस्। सही ढाँचामा फाइल उत्पन्न गर्न सजिलो बाटो यसलाई निर्यात गरि , आवश्यक परिवर्तन गरेर पुन:आयात गर्नु हो।
+Translations = अनुवादन
 Unread\ below = पढ्न बाँकी सन्देश तल
 Unsaved\ changes = बचत नभएका परिवर्तनहरू
 Unverified = रुजु नगरिएको
 Unverify = रुजु नगर्न
 Update = अद्यावधिक
 Update\ Facility = संस्था परिमार्जन
-Upload\ Forms = 
+Upload\ Forms = फर्म अपलोड गर्नुहोस्
```
[the remark this answers is not in the document]
Comment (1yuv) [reply api/resources/translations/messages-ne.properties:263]: धन्यवाद। `फर्म` प्रयोग गर्ने।

File: api/resources/translations/messages-ne.properties [hunk 37]
```diff
@@ -279,34 +279,35 @@ Visits = स्वाथ्य संस्थामा जाँच
 Weeks\ Pregnant = गर्भवती भएको हप्ता
 _id = रेकर्ड UUID
 about = बारे
+about.cht = Powered by
```
[the remark this answers is not in the document]
Comment (1yuv) [reply api/resources/translations/messages-ne.properties:282]: It will read as `द्वारा संचालित X` if we translate, so I left as this.

**Review by binokaryg — APPROVED** [review]
Thank you for your hard work on the translations. This is a significant task, and your efforts will have a lasting impact.

**Pull request merged**